docs/UserGuide.md

```diff
@@ -155,38 +155,85 @@ This command removes the module stated in the command (“CS2103T”) from the l
 
 ### Tagging a module to contact : `tag`
 
-Adds a module tag to an existing contact.
-Format: `tag n/NAME m/MODULE_TAG`
-Example of usage: `tag n/John Doe m/CS2103T`
+Adds module tag(s) to an existing contact.
+
+Formats: 
+* `tag CONTACT_INDEX m/MODULE_TAG`
+* `tag m/MODULE_TAG`
+
+Example of usage: `tag 3 m/CS2103T`
 ```
 Name: John Doe (User)
-Modules reading: CS2101 | MA2104 | MA3252 | CFG1002
+Modules: [CS2101 , MA2104 , MA3252 , CFG1002]
+```
+
+Expected outcome for CLI:
+```
+Module(s) tagged to Person!
+Name: John Smith
+Modules: [CS2100, CS2101, CS2102, CS2103T]
+Module(s) in common: [CS2101, CS2103T]
+```
+Description of outcome: 
+
+CS2103T is added to John Doe's list of modules. Assuming the user also takes CS2101 and CS2103T, which are represented as the modules in common.
+
+Example of usage: tag m/CS2103T
+```
+Name: John Doe (user)
+Modules: [CS2101, MA2104, MA3252, CFG1002]
 ```
 
 Expected outcome for CLI:
 ```
+Module(s) tagged to Person!
 Name: John Doe
-Modules: CS2100 | CS2101 | CS2102 | CS2103T
-1 Common Module: CS2101 | CS2103T
+Modules: [CS2101, CS2103T, MA2104, MA3252, CFG1002]
 ```
-Description of outcome: CS2103T is added to John Doe's list of modules. Assuming the user also takes CS2101 and CS2103T, which are represented as the modules in common.
+Description of outcome:
+
+CS2103T is added to John Doe's, the user, list of modules.
 
 ### Untagging a module from an existing contact : `untag`
 
 Removes a module tag from an existing contact.
-Format: `untag n/NAME m/MODULE_TAG`
 
-Example of usage: `untag n/John Doe m/CS2103T`
-Context:
+Formats: 
+* `untag CONTACT_INDEX m/MODULE_TAG`
+* `untag m/MODULE_TAG`
+
+Example of usage: `untag 3 m/CS2103T`
+```
+Name: John Doe (User)
```

Comment:
As per CS2101 recommendations, lets not use John Doe and change the name up abit.